pkg/operator/ceph/test/containers.go

```diff
@@ -58,10 +58,10 @@ func (ct *ContainersTester) AssertArgsContainCephRequirements() {
 			continue // don't consider containers that aren't Ceph commands
 		}
 		requiredFlags := []string{
-			"--log-to-stderr=true",
-			"--err-to-stderr=true",
-			"--mon-cluster-log-to-stderr=true",
-			"--log-stderr-prefix=debug ",
+			"--default-log-to-stderr=true",
```

Comment:
I am not sure how users can override these settings if they so desire.
As requested

Reply:
Hi, as I understand in the issue description, some users wants to use options with `--default-*`

> Some users have wished to change these options for whatever reasons in the past. With `--default-*` versions, Rook can set CLI options that will ensure the average user will still get all logs to stderr as they would expect, but advanced users could enjoy different behavior.

This change should be enough, but  I'm not sure if I'm missing something